docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -884,6 +885,57 @@ _Available in Fleet Premium_
 
 `Status: 200`
 
+### Bulk delete hosts by filter or ids
+
+`POST /api/v1/fleet/hosts/delete`
+
+#### Parameters
+
+| Name    | Type    | In   | Description                                                                                                                                                                                                                                                                                                                        |
+| ------- | ------- | ---- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ids     | list    | body | A list of the host IDs you'd like to delete. If `ids` is specified, `filters` cannot be specified.                                                                                                                                                                                                                                                           |
+| filters | object  | body | Contains any of the following four properties: `query` for search query keywords. Searchable fields include `hostname`, `machine_serial`, `uuid`, and `ipv4`. `status` to indicate the status of the hosts to return. Can either be `new`, `online`, `offline`, or `mia`. `label_id` to indicate the selected label. `team_id` to indicate the selected team. If `filters` is specified, `id` cannot be specified.  |
```

Comment:
Should also specify that the `label_id` cannot be specified with the `status` field, if I read `hostIDsFromFilters` correctly?